### PR TITLE
{bio}[foss/2015b] CD-HIT 4.6.1, CD-HIT 4.6.4 (OK)

### PR DESCRIPTION
**Pull request by nathanhaigh**
Working on Easyconfigs for CD-HIT-4.6.4-foss-2015b

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.4-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = "CD-HIT"
+version = "4.6.4"
+versionsuffix = "-2015-0603"
```
Comment (boegel) [inline easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.4-foss-2015b.eb]: filename should be adjusted accordingly: `CD-HIT-4.6.4-foss-2015b-2015-0603.eb`

File: easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.1-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = "CD-HIT"
+version = "4.6.1"
+versionsuffix = "-2012-08-27"
```
Comment (boegel) [inline easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.1-foss-2015b.eb]: filename should be adjusted accordingly

**Comment by nathanhaigh** [comment]
Test report by @nathanhaigh
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux ubuntu 14.04, Intel(R) Xeon(R) CPU E5-2699 v3 @ 2.30GHz, Python 2.7.6
See https://gist.github.com/5a5005d8cc454ab1f0a8 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2eba232fef9f0e6ee1ae for a full test report.

**Comment by boegel** [comment]
Going in, thanks @nathanhaigh

**Pull request merged**